views/Graph.react.js

```diff
@@ -390,54 +483,94 @@ class Graph extends Component {
     socket.close()
     window.removeEventListener('resize', this.handleResize)
   }
-
+  updateUserPreferences = (configuration) => {
+    let uid = this.props.user.uid
+    const createNewPreference = preparePreferences(
+      configuration,
+      this.state.preferences
+    )
+    return window
+      .fetch(`${basePath}/api/v1/users/${uid}/preferences`, {
```

Comment:
can we use an API route for this?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/129/commits/75fa4e051ac01363f4567f566d2f67ddf2280f34